### PR TITLE
Change readability of header rows in PDF; tweak slack_report to not actually send report if --nosend specified, etc

### PR DESCRIPTION
**Pull request by royrapoport**

### CODE REVIEW

File: templates/general.html [hunk 1]
```diff
@@ -72,19 +72,19 @@
                 border-width:0px 0px 0px 0px;
             }
             .CSSTableGenerator tr:first-child td{
-                background:-o-linear-gradient(bottom, #4c4c4c 5%, #000000 100%);	background:-webkit-gradient( linear, left top, left bottom, color-stop(0.05, #4c4c4c), color-stop(1, #000000) );	background:-moz-linear-gradient( center top, #4c4c4c 5%, #000000 100% );	filter:progid:DXImageTransform.Microsoft.gradient(startColorstr="#4c4c4c", endColorstr="#000000");	background: -o-linear-gradient(top,#4c4c4c,000000);
-                background-color:#4c4c4c;
+                background:-o-linear-gradient(bottom, #ddd52a 5%, #ffffff 100%);	background:-webkit-gradient( linear, left top, left bottom, color-stop(0.05, #ddd52a), color-stop(1, #ffffff) );	background:-moz-linear-gradient( center top, #ddd52a 5%, #ffffff 100% );	filter:progid:DXImageTransform.Microsoft.gradient(startColorstr="#ddd52a", endColorstr="#ffffff");	background: -o-linear-gradient(top,#ddd52a,ffffff);
```
Comment (jdq22) [inline templates/general.html:75]: Will you please drop in a screenshot so we can easily see how the readability was changed? (I'm assuming the change was in the more readable direction ;))

File: slack_report.py [hunk 2]
```diff
@@ -25,10 +25,13 @@
 pdf = pdf_formatter.PDFFormatter()
 
 send = not args.nosend
-if not args.destination:
-    raise RuntimeError("Must specify --destination")
-destination = report_utils.override(args.destination)
-print("Will send report to {}".format(destination))
+if send and not args.destination:
+    raise RuntimeError("Must specify --destination if did not specify --nosend")
```
Comment (jdq22) [inline slack_report.py:29]: What are the expected inputs for `--destination`?

**Review by jdq22 — CHANGES_REQUESTED** [review]

File: templates/general.html (hunk 1 above)
Comment (royrapoport) [reply templates/general.html:75]: image

Comment (royrapoport) [reply templates/general.html:75]: image

File: slack_report.py (hunk 2 above)
Comment (royrapoport) [reply slack_report.py:29]: Either @username or #channel-name

**Review by jdq22 — APPROVED** [review]

**Pull request merged**